bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/FindReplaceLogic.java

```diff
@@ -497,20 +514,19 @@ private Point replaceSelection(String replaceString, boolean regExReplace) {
 	 * Returns whether the specified search string can be found using the given
 	 * options.
 	 *
-	 * @param findString             the string to search for
 	 * @return <code>true</code> if the search string can be found using the given
 	 *         options
 	 *
 	 */
-	private boolean findNext(String findString) {
+	private boolean findNext(boolean updateFromExistingBaseLocation) {
```

Comment:
```suggestion
	private boolean findNext(boolean updateFromIncrementalBaseLocation) {
```

to be consistent with the variable called `incrementalBaseLocation`

Reply:
Fixed in 9978946